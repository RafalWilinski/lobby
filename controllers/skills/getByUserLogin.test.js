const getByUserLogin = require("./getByUserLogin");

describe("GET /user/skills", () => {
  it("should return 200", async () => {
    const ctx = {
      request: {
        query: {
          userLogin: 'testowy@test.pl'
        }
      }
    };

    await getByUserLogin(ctx);

    expect(ctx.body).toMatchSnapshot();
  });
});
