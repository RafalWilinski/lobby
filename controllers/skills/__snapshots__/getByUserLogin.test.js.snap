// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GET /user/skills should return 200 1`] = `
Object {
  "data": Array [],
}
`;
